cli/src/graphar_cli/importer.py

```diff
@@ -0,0 +1,122 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from logging import getLogger
+
+from .config import ImportConfig
```

Comment:
I thing it can be moved to the `TYPE_CHECKING` block, isn't? I do not see runtime calls to this class

Reply:
Can you provide some examples of using `TYPE_CHECKING`? It would help me a lot, thanks!

Reply:
https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING

Reply:
> https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING

Thanks! I will try it with the `protobuf` changes.